addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -1,7 +1,12 @@
 @include media-breakpoint-down(md) {
     img, .media_iframe_video, span.fa, i.fa {
         &:not(.o_animate) {
-            transform: none !important;
+            // For RTL (Right-to-Left) purposes, don't disable the transform directive for 
+            // Font Awesome icons that are flipped by 180deg.
+            // Exclude elements with classes containing 'fa' and ending with '-left' or '-right'.
+            &:not([class~="fa"][class*="-left"]):not([class~="fa"][class*="-right"]) {
+                transform: none !important;
+            }
```

Comment:
Could you not nest this new `:not()` condition? Eg if we add another case later, it will be indent-ception :)

About this whole `transform: none !important;` which after some digging I found was introduced with https://github.com/odoo/odoo/commit/88eaa5f26fb7a44250eefa3abeb5d14b23559289 10 years ago, I am wondering if we really want to keep it at all (in master ofc)..
Is there a really good reason to disable the transform in mobile, at least on icons? For images I guess our transform option on image (which should be in debug imo) make sense to disable in mobile (maybe not so much tho..)

cc @qsm-odoo Maybe on master we could just remove at least the `.fa` part (which is not even considering odoo icons anyway) from this selector ?
Removing the whole rule might be something to consider too

Reply:
> which after some digging I found was introduced with https://github.com/odoo/odoo/commit/88eaa5f26fb7a44250eefa3abeb5d14b23559289 10 years ago

Oups :)

Reply:
@qsm-odoo I understood for `img:not(...)` but don't I have to target all fa-...-right and fa-...-left instead of targeting only fa-chevron-right and fa-chevron-left? see this snippet : https://github.com/odoo/odoo/commit/7e2f9634d256918e18bbc994063bf70bf1aa874e#diff-89e948db95cbda99cb06e31e2107e107c103716598c755c284e2bb1a4d5ea54dR3-R36

Reply:
@qsm-odoo or maybe this selector to avoid copy and paste all the .fa-...-left and .fa-...-right : 
```[class^="fa-"]:not([class$="-left"]):not([class$="-right"]) {
...
}```

Reply:
This selector will match nothing as you search for elements whose class start with `fa-`. And in practice `[class^="fa-"]` is equal to `.fa` because there is no element with a `fa-XXX` class without the class `fa`. And even then your selector would still match `class="fa-home fa padding-left"` which is not what you want: the only reliable way is indeed to list the classes (would be nice to share that list in a variable if this is really needed).

Reply:
The `[class]` selector is about the whole className, not each individual class. And if you combine two, it's just two different selectors.

Reply:
@qsm-odoo Ok, that's what I thought when discussing about it with @rdeodoo . I'll do the necessary job then. Thanks

Reply:
@qsm-odoo I've adapted the SCSS as you suggested.